$OpenBSD: patch-CMakeLists_txt,v 1.1.1.1 2013/04/24 08:42:12 zhuk Exp $
Force using more strict rules, to avoid potential problems
with CMake module names clashing.
--- CMakeLists.txt.orig	Sat Jul 30 19:49:26 2011
+++ CMakeLists.txt	Mon Feb  6 02:00:22 2012
@@ -1,6 +1,6 @@
 project(PhononVLC)
 
-cmake_minimum_required(VERSION 2.6.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 2.8.7 FATAL_ERROR)
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 
 # Phonon
